application/src/main/java/org/togetherjava/tjbot/commands/moderation/modmail/ModMailCommand.java

```diff
@@ -43,7 +48,7 @@ public final class ModMailCommand extends SlashCommandAdapter {
 
     private static final Logger logger = LoggerFactory.getLogger(ModMailCommand.class);
     public static final String COMMAND_NAME = "modmail";
-    private static final String OPTION_MESSAGE = "message";
+    private static final String MESSAGE = "message";
```

Comment:
it has to be clear what this field is and where its used. the current name doesnt tell.
maybe go for `MODAL_INPUT_MESSAGE` or similar